### PR TITLE
Allow non subadmins to change language

### LINKED ISSUE #2568: user can't change language in personal settings
A normal user can't change the language in the personal settings (for admins it works).

The user gets a 403 but the method is set to "@NoAdminRequired". After some debugging I found out that here: https://github.com/nextcloud/server/blob/master/lib/private/AppFramework/App.php#L91 the "PersonalController" is not found, but only for non-admins.

Any ideas? Maybe @rullzer as the original author of the controller?

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #2568

### CODE REVIEW

**Comment by rullzer** [comment]
A bitten by my own annotations... LGTM

File: settings/Controller/PersonalController.php [hunk 1]
```diff
@@ -69,6 +69,7 @@ public function __construct($appName,
 
 	/**
 	 * @NoAdminRequired
+	 * @NoSubadminRequired
```
Comment (nickvergessen) [inline settings/Controller/PersonalController.php]: @rullzer why is this required on this controller, and where is defined to which controllers the SubadminMiddleware reacts. Because it seems this is too greedy now, but e.g. activities and notifications still work without the subadmin thing

**Review by rullzer — APPROVED** [review]

File: settings/Controller/PersonalController.php (hunk 1 above)
Comment (rullzer) [inline settings/Controller/PersonalController.php]: It is just in the settings middleware. Security stuff should be opt-out not opt-in.

Comment (nickvergessen) [inline settings/Controller/PersonalController.php]: Yeah well what I ment is make sure it only reacts to Settings controllers? Currently it reacts to all controllers once loaded, and this only requires one OC\Settings class to be called?

**Comment by schiessle** [comment]
works. 👍

File: settings/Controller/PersonalController.php (hunk 1 above)
Comment (rullzer) [inline settings/Controller/PersonalController.php]: Ah that. Mmmmm let me check

Comment (rullzer) [inline settings/Controller/PersonalController.php]: Ah no it does. middleware is per app.
